content/v2.0/get-started.md

```diff
@@ -194,6 +195,100 @@ docker exec -it influxdb /bin/bash
 {{% /tab-content %}}
 <!--------------------------------- END Docker -------------------------------->
 
+<!-------------------------------- BEGIN kubernetes---------------------------->
+{{% tab-content %}}
+
+You can install InfluxDB in a local kubernetes environment with Minikube.
```

Comment:
I'd update this to say you can install InfluxDB in a Kubernetes Cluster. Below you will find instructions for using Minikube, but the steps should be similar in any Kubernetes cluster.